docker-hub/publish/insights-analytics.md

```diff
@@ -0,0 +1,127 @@
+---
+title: Insights & analytics
```

Comment:
Do we want to add anything to the DVP overview page that references/links to this page?

Reply:
I will update the DVP overview page in a separate PR, so I will make sure to include a link to this page then